source/guides/maintenance/fix-broken-install.rst

```diff
@@ -43,21 +43,22 @@ Mount root partition, verify, and fix
 
 #. Ensure the system is connected to the network.
 
-#. Mount the system’s root partition.
+#. Mount the system’s root and boot partitions.
```

Comment:
swupd does not manage the EFI partition.  So we it's not necessary to mount the boot partition.